tests/services-aerospace.nix

```diff
@@ -31,6 +31,6 @@ in
       ${config.out}/user/Library/LaunchAgents/org.nixos.aerospace.plist`
 
     echo >&2 "checking config in $conf"
-    if [ `cat $conf | wc -l` -eq "27" ]; then echo "aerospace.toml config correctly contains 27 lines"; else return 1; fi
+    if [ `cat $conf | wc -l` -eq "29" ]; then echo "aerospace.toml config correctly contains 29 lines"; else return 1; fi
```

Comment:
Can you replace this check with something that actually checks the contents of `$conf` to see that it is setting the settings correctly like:

https://github.com/LnL7/nix-darwin/blob/c6b65d946097baf3915dd51373251de98199280d/tests/services-skhd.nix#L20

Reply:
i was doubtful of the initial implementation of the testing as well, unfortunately I didn't know much about it, I'll try this weekend to replicate the skhd one.